aip/general/0213.md

```diff
@@ -18,24 +18,105 @@ into taking major version changes or even lead to dependency conflicts.
 
 However, there are also cases where common structures are valuable, especially
 where a concept is well-known and it is sufficiently clear that it will not
-change. Common protos serve this use case.
+change. Common protos serve this use case. There are two forms of common protos:
+organization-specific and global.
 
-## Guidance
+## Guidance for API protos
```

Comment:
I'd switch this back. Every other AIP has a heading "Guidance" - it's just the standard template.

Reply:
Let's chat about the *structure* of this change in our 1:1 later today. Fundamentally, there are three sections:

- API protos
- Organization-specific common protos
- Global common protos

I fear that combining all of those together in terms of "all guidance, then all rationale" will make it harder to navigate. But maybe that means this should be two AIPs after all.

As I say, let's chat :)